web/rootfs/defaults/nginx.conf

```diff
@@ -33,7 +35,11 @@ http {
 	# Logging Settings
 	##
 
+	{{ if .ENABLE_ACCESS_LOGS }}
 	access_log /dev/stdout;
+	{{ else }}
+	access_log off;
+	{{ end }}
 	error_log /dev/stderr;
```

Comment:
do we also want to disable error logging perhaps?

Reply:
Maybe add custom log levels for the web containers as well? Info would include access logs, warn or error would include error logs, and none/false would deactivate both.

Reply:
I have added support for more levels

Reply:
what do you think? Is that useful?